pulpcore/cli/common.py

```diff
@@ -75,12 +75,12 @@ def _config_callback(ctx, param, value):
 )
 @click.option("--base-url", default="https://localhost", help="Api base url")
 @click.option("--user", help="Username on pulp server")
-# @click.option("--password", prompt=True, hide_input=True, help="Password on pulp server")
 @click.option("--password", help="Password on pulp server")
 @click.option("--verify-ssl/--no-verify-ssl", default=True, help="Verify SSL connection")
-@click.option("--format", type=click.Choice(["json"], case_sensitive=False), default="json")
+@click.option("--format", type=click.Choice(["json", "none"], case_sensitive=False), default="json")
```

Comment:
What does `none` do here?

Reply:
Oh, is't the best feature you get for 8 added characters. Nothing. (It supresses normal output instead of json-rendering it.)